xmlChart.go

```diff
@@ -380,6 +380,7 @@ type cChartLines struct {
 // cScaling directly maps the scaling element. This element contains
 // additional axis settings.
 type cScaling struct {
+	LogBase     *attrValString `xml:"logBase"`
```

Comment:
The data type of `logBase` should be an integer, by [OOXMLSchemas](https://github.com/sc34wg4/OOXMLSchemas/blob/2b074ca2c5df38b18ac118646b329b508b5bdecc/Part1/OfficeOpenXML-XMLSchema-Strict/dml-chart.xsd#L1142-L1147), I propose that change its type to `attrValInt `, and check its min and max value (2-1000). in addition, could you add the unit testing case for the changes?

Reply:
Thank you very much for the suggestion, I couldn't find quickly where to look for the standard definition so just refer from a real XLSX file without knowing the type or value range.

I have updated the code accordingly and added test cases as well.